### PR TITLE
kubelet: Clean up computePodContainerChanges.

### PR DESCRIPTION
**Pull request by yifan-gu**
More `continue`, less `else` to make it less indent and more readable.

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1107,51 +1107,52 @@ func (kl *Kubelet) computePodContainerChanges(pod *api.Pod, runningPod kubeconta
...
+				continue
+			}
+			if result == probe.Success {
+				glog.V(4).Infof("probe success: %q", container.Name)
+				containersToKeep[containerID] = index
+				continue
+			}
+			glog.Infof("pod %q container %q is unhealthy (probe result: %v), it will be killed and re-created.", podFullName, container.Name, result)
+			containersToStart[index] = empty{}
+		} else { // createPodInfraContainer == true and Container exists
```
Comment (vmarmol) [inline pkg/kubelet/kubelet.go]: We can probably move this one out too and add a continue.

**Comment by vmarmol** [comment]
+1, I am partial to this pattern as well.

**Comment by yifan-gu** [comment]
@vmarmol Comments addressed. e2e passed!

```
Ran 35 of 40 Specs in 1051.233 seconds
SUCCESS! -- 35 Passed | 0 Failed | 1 Pending | 4 Skipped I0415 10:10:12.011448    9228 driver.go:117] All tests pass
2015/04/15 10:10:12 Running: teardown
Project: robust-chess-87305
```

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**